extensions/wikia/AdEngine/js/AdSlotsVenus.js

```diff
@@ -2,127 +2,13 @@
 require([
 	'jquery',
 	'wikia.document',
-	'wikia.log',
-	'wikia.window',
-	'ext.wikia.adEngine.adPlacementChecker'
-], function ($, doc, log, win, adPlacementChecker) {
+	'ext.wikia.adEngine.adSlotsInContent'
+], function ($, document, adSlotsInContent) {
```

Comment:
We're not reusing names that are available in the global context, thus using doc for AMD module, not document.